sql/1.095/github_user.sql

```diff
@@ -0,0 +1,8 @@
+BEGIN;
+    CREATE TABLE github_user_link (
+        "users_id" bigint NOT NULL,
+        "data" text NOT NULL
+    );
+
+    ALTER TABLE users ADD COLUMN "github_user" text;
+COMMIT;
```

Comment:
@jbarrett Ah, right, for some reason I thought it only existed on wing and not in production :)
Removed now!